styles/scss/components/menu/menu-overlay.scss

```diff
@@ -0,0 +1,65 @@
+.video-react-options-close {
+  position: absolute;
+  top: 0.4em;
+  right: 0.5em;
+  color: white;
+  font-size: 2.5em;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+}
+
+.video-react-options-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 325px;
+  background: rgba(#000000, 0.8);
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Circular", sans-serif;
```

Comment:
Non-blocking feedback. Something I know we talked about in person, but seems worth documenting. I'd love to, now or eventually, use our Angelou font constants here. I expect that means structuring the overlay component so that we can pass it style configuration from the "outside" (that is, from the Angelou video player).

Reply:
Set this up to do that now.